.editorconfig

```
@@ -35,6 +35,11 @@ dotnet_style_parentheses_in_other_operators = never_if_unnecessary:silent
 # Modifier preferences 
 dotnet_style_require_accessibility_modifiers = for_non_interface_members:silent 
 dotnet_style_readonly_field = true:suggestion 
+
+# Code-block preferences
+csharp_prefer_braces = when_multiline:suggestion
+csharp_prefer_simple_using_statement = true:suggestion
```

This does not work with unity,  this is a new feature of C# 8 that is not available yet
```suggestion
csharp_prefer_simple_using_statement = false
```